cli/doc/parser.rs

```diff
@@ -193,6 +198,77 @@ impl DocParser {
     Ok(flattened_docs)
   }
 
+  fn get_doc_nodes_for_module_imports(
+    &self,
+    module_body: Vec<swc_ecmascript::ast::ModuleItem>,
+    referrer: &str,
+  ) -> Result<Vec<DocNode>, ErrBox> {
+    let mut imports = vec![];
+
+    for node in module_body.iter() {
+      match node {
+        swc_ecmascript::ast::ModuleItem::ModuleDecl(module_decl) => {
+          match module_decl {
+            ModuleDecl::Import(import_decl) => {
+              let (js_doc, location) = self.details_for_span(import_decl.span);
+              for specifier in &import_decl.specifiers {
+                use swc_ecmascript::ast::ImportSpecifier::*;
+
+                let (name, maybe_imported_name, src) = match specifier {
+                  Named(named_specifier) => (
+                    named_specifier.local.sym.to_string(),
+                    named_specifier
+                      .imported
+                      .as_ref()
+                      .map(|ident| ident.sym.to_string()),
```

Comment:
Could you add a `.or_then(|| Some(named_specifier.local.sym.to_string()))` to this? There is no way to tell the difference between this and a namespace import otherwise :-)